x/cork/handler.go

```diff
@@ -10,30 +10,32 @@ import (
 	"github.com/peggyjv/sommelier/v4/x/cork/types"
 )
 
-// NewHandler returns a handler for "oracle" type messages.
+// NewHandler returns a handler for "cork" type messages.
```

Comment:
I'm not sure if the legacy handler is even required with the default GRPC handler any more.